docs/wallet_API.md

```diff
@@ -244,7 +254,7 @@ Although it is possible and perfectly fine to transfer tokens directly from the
 
 ### - Transfer between implicit accounts
 
-```js live noInline wallet
+```js live noInline
```

Comment:
For live examples using beacon wallet, we need to add the props `wallet` otherwise an ephemeral key is used to sign:
```js live noInline wallet